fscollections/migrations/0011_collection_sounds.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.2.19 on 2025-03-31 10:50
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sounds', '0055_remove_sound_geotag'),
+        ('fscollections', '0010_remove_collection_max_num_sounds'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='collection',
+            name='sounds',
+            field=models.ManyToManyField(blank=True, related_name='collections', through='fscollections.CollectionSound', to='sounds.sound'),
+        ),
```

Comment:
I think we should probably consolidate all these migrations into a single one. I can try and do that when testing this PR.